src/pybroker/log.py

```diff
@@ -57,6 +57,16 @@ def _update_progress_bar(self, count: int):
             self._progress_bar = None
             self._out("")
 
+    @property
+    def progress_bar_flag(self):
```

Comment:
Can we reuse the existing fields instead? Also, not sure why these are negated based on the names.

Reply:
Can you change the property names to:

progress_bar_disabled
disabled

And also remove the negation?